docs/Project-Configuration/Best-Practices.md

```diff
@@ -29,7 +30,7 @@ Use [task tags](Project-Configuration-Files#task-and-variant-tags) to reduce rep
 
 ## Expansions
 
-Be cautious about Evergreen's expansion syntax.
+Be cautious about Evergreen's [expansion syntax](../Project-Configuration/Project-Configuration-Files#expansions).
```

Comment:
Since the two files are in the same sub-directory, the link should be just `Project-Configuration-Files#expansions`